ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -2345,6 +2345,19 @@ def get_sssd_version(host):
     return parse_version(version)
 
 
+def get_healthcheck_version(host):
+    """
+    Function to get healthcheck version on fedora and rhel
+    """
+    platform = get_platform(host)
+    if platform == 'fedora':
+        cmd = host.run_command(['rpm', '-q', 'freeipa-healthcheck'])
+    else:
+        cmd = host.run_command(['rpm', '-q', 'ipa-healthcheck'])
```

Comment:
you can even use `rpm -qa --qf %{VERSION} *ipa-healthcheck`
it prints only the version (without package name or release) and may be enough for you to compare:
on fedora:
```
# rpm -qa --qf %{VERSION} *ipa-healthcheck
0.6
```
on redhat:
```
# rpm -qa --qf %{VERSION} *ipa-healthcheck
0.4
```

Reply:
It just isn't cross-platform at all. It seems like we should have an ipaplatform task for is_package_installed() or something like it.

The base perhaps would have rpm -q and other distros could override as necessary.

Reply:
We already have this broken in ipatests/test_integration/test_epn.py and ipatests/test_integration/test_installation.py. At the same time, in `tasks.py` we have `intall_packages` that already supports both RPM and APT based distributions.

Reply:
I believe at least part of the epn rpm usage is due to repo problems we were having during development and it just got left over. IMHO it can be removed, what do you think @fcami.

As for test_installation, yes, it should probably have a decorator to run only on Fedora/RHEL.

My purpose to bring this up is to avoid distro-specific usages spreading further into the tests which will only make it harder later to remove it.